SConstruct

```diff
@@ -52,14 +52,14 @@ if env['debug']:
 # This command generates the header with git revision information
 def gen_gitrev(env, target, source):
 	revid = subprocess.check_output(["git", "rev-parse", "HEAD"]);
-	fulltag = subprocess.check_output(["git", "tag", "--sort=v:refname"]).split('\n')[-1]
+	fulltag = subprocess.check_output(["git", "tag", "--sort=v:refname"]).decode('utf-8').split('\n')[-1]
 	tagrev = subprocess.check_output(["git", "rev-parse", fulltag]) if fulltag else ""
 	with open(target[0].path, 'w') as gitrev_hpp:
-		print >>gitrev_hpp
-		print >>gitrev_hpp, '#define GIT_REVISION "' + revid[0:7] + '"'
-		print >>gitrev_hpp, '#define GIT_TAG "' + fulltag + '"'
-		print >>gitrev_hpp, '#define GIT_TAG_REVISION "' + tagrev[0:7] + '"'
-		print >>gitrev_hpp
+		print(gitrev_hpp)
```

Comment:
I believe that this change may be incorrect? My understanding was that Python2 `print >>xxx` maps to Python3 `print(file=xxx)`.

Reply:
Yes, you are right, will fix.

Reply:
Fixed

Reply:
Not really, it breaks Mac OS build.